OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/URLInterceptorWebViewClient.java

```diff
@@ -179,6 +180,12 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         }
     }
 
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
```

Comment:
We should mark deprecated the following function
`public boolean shouldOverrideUrlLoading(WebView view, String url)`

Refactor the functionality of this deprecated function into another non-overridden function 
and call that function from both new and deprecated functions.